apps/dashboard/src/components/workflow-editor/steps/digest/scheduled-digest.tsx

```diff
@@ -0,0 +1,168 @@
+import { useMemo } from 'react';
+import cronParser from 'cron-parser';
+
+import {
+  getCronBasedOnPeriod,
+  getPeriodFromCronParts,
+  parseCronString,
+  PeriodValues,
+  toCronFields,
+  toUiFields,
+  UiCronFields,
+} from '@/components/workflow-editor/steps/digest/utils';
+import { Period } from '@/components/workflow-editor/steps/digest/period';
+import { NumbersPicker } from '@/components/workflow-editor/steps/digest/numbers-picker';
+import { DaysOfWeek } from '@/components/workflow-editor/steps/digest/days-of-week';
+import { MultiSelect } from '@/components/primitives/multi-select';
+
+const MONTHS_OPTIONS = [
+  { value: 1, label: 'January' },
+  { value: 2, label: 'February' },
+  { value: 3, label: 'March' },
+  { value: 4, label: 'April' },
+  { value: 5, label: 'May' },
+  { value: 6, label: 'June' },
+  { value: 7, label: 'July' },
+  { value: 8, label: 'August' },
+  { value: 9, label: 'September' },
+  { value: 10, label: 'October' },
+  { value: 11, label: 'November' },
+  { value: 12, label: 'December' },
+];
+
+export const ScheduledDigest = ({
```

Comment:
This is the main scheduled digest component that parses and generates the cron value based on the form fields.